extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -215,16 +237,18 @@ export function initializeWizardPage(context: WizardPageContext): void {
 
 export function createSection(context: SectionContext): azdata.GroupContainer {
 	const components: azdata.Component[] = [];
-	context.sectionInfo.inputWidth = context.sectionInfo.inputWidth || DefaultInputComponentWidth;
-	context.sectionInfo.labelWidth = context.sectionInfo.labelWidth || DefaultLabelComponentWidth;
+	context.sectionInfo.inputWidth = context.sectionInfo.inputWidth || DefaultInputWidth;
+	context.sectionInfo.labelWidth = context.sectionInfo.labelWidth || DefaultLabelWidth;
+	context.sectionInfo.rowAlignItems = context.sectionInfo.rowAlignItems || DefaultRowAlignItems;
+	context.sectionInfo.rowWidth = context.sectionInfo.rowWidth || DefaultRowWidth;
+	context.sectionInfo.rowHeight = context.sectionInfo.rowHeight || DefaultRowHeight;
+	context.sectionInfo.inputWidth = context.sectionInfo.inputWidth || DefaultInputWidth;
```

Comment:
You're duplicating the inputWidth and labelWidth here

Reply:
good catch will remove the duplication.

Reply:
fixed